proxy.go

```diff
@@ -180,12 +181,26 @@ func (proxy *ProxyHttpServer) ServeHTTP(w http.ResponseWriter, r *http.Request)
 		if origBody != resp.Body {
 			resp.Header.Del("Content-Length")
 		}
-		copyHeaders(w.Header(), resp.Header, proxy.KeepDestinationHeaders)
```

Comment:
wouldn't removing this fail to copy other important headers? like cookies, etc?

Reply:
Hmm I think there was an issue where this wasn't writing the headers at all (hence the change at https://github.com/Windscribe/goproxy/pull/2/files#diff-fd4820e2c415e0057b0e67d294303bdde3b98c2731b6dbd1e2b0f79349744499R190), but I think you're correct in that this new change still isn't writing this the way we want, taking another look.